client/src/components/RatingsAndReviews/Reviews/FormIstics.jsx

```diff
@@ -13,17 +16,24 @@ export default function IsticsForm({istics}) {
   }
 
   return (
-    <>
+    <div>
     {Object.keys(istics).map(istic => (
-      <>
-      <h2>{istic}</h2>
-      <select name={istic}>
-        {attributes[istic].map(attr => (
-          <option value={attr}>{attr}</option>
-        ))}
-      </select>
-      </>
+      <Characteristic key={istics[istic].id}>
+        <label htmlFor="istic">
+          {istic}:
+        </label>
+        <select onChange={onChange} id={istics[istic].id} name={istic} defaultValue="empty" required>
+          <option id ="empty" value="" isdisabled="true">
+            Select your option
+          </option>
+          {attributes[istic].map((attr, idx) => (
+            <option id={idx} key={attr} value={idx}>
+              {attr}
+            </option>
+          ))}
+        </select>
+      </Characteristic>
     ))}
-    </>
+    </div>
```

Comment:
fragment tags should be fine, probably better to use them instead of unnamed div